pyfolio/timeseries.py

```diff
@@ -31,6 +28,9 @@
 from .interesting_periods import PERIODS
 
 
+#####################################
```

Comment:
This indicates to me that we should split this into a risk.py module here or pull the non-risk metric code into another module.